port_ocean/tests/core/handlers/entity_processor/test_jq_entity_processor.py

```diff
@@ -269,3 +269,35 @@ async def test_parse_items_performance_10000(
             assert len(result.entity_selector_diff.passed) == 1
             assert result.entity_selector_diff.passed[0].properties.get("foo") == "bar"
             assert not result.errors
+
+    async def test_parse_items_wrong_mapping(
```

Comment:
can you add a test for logging the missing required fields?

Reply:
I wanted to avoid monitoring the logs themselves so instead i've asserted for the assumed misconfigurations. 
in this case I assume that the log will be written properly. 
assert result.misconfigured == {
            "identifier": ".ark",
            "foo": ".bazbar",
            "desc": ".foobar",
            "name": ".bar.baz",
        }
        
@Tankilevitch, do you want to make sure that in the future the text message won't change? this is why you want the log to be tested as well?